src/cpp/processquit.cpp

```diff
@@ -74,8 +76,12 @@ class SafeSocketNotifier : public QObject
 public slots:
 	void setEnabled(bool enable)       { sn->setEnabled(enable); }
 
-signals:
-	void activated(int socket);
+	void doActivated(){
+		activated(sn->socket());
+	}
+
```

Comment:
Let's move this to a `private slots:` section at the end.

Also, opening `{` should be on the next line (unless written compactly like `func() { ... }`, which would be acceptable here too).